DEPENDENCIES

```diff
@@ -202,6 +202,83 @@ Prerequisites for building the OpenDataPlane (ODP) API
    1024MB of memory:
    $ sudo ODP_PKTIO_DPDK_PARAMS="-m 1024" ./test/performance/odp_l2fwd -i 0 -c 1
 
+3.5 Code instrumentation with PAPI library (optional)
+
+   PAPI library is used to instrument ODP API.
+
+3.5.1 Building PAPI library
+
+   ODP example library 'instrum' is using PAPI library to retrieve performance
+   counters associated with execution of ODP API. PAPI library is currently at 5.6.0.
+   However, if possible one should try to use the latest PAPI master branch commit for
+   the best performance and the latest bug fixes.
+
+   # Checkout PAPI code
+   $ git clone https://bitbucket.org/icl/papi.git
+   $ cd papi
+   $ git checkout papi-5-6-0-t (optional)
+
+   # Build PAPI
+   $ cd ./src/
+   $ ./configure --prefix=<papi_install_dir>
+   $ make clean
+   $ make
+   $ make install
```

Comment:
Isn't apt-get install libpapi-dev / yum install / etc. enough?

Reply:
Don't know. I was not able to get it for my Ubuntu. You should get something like 'libpfm4 libpapi5.4 libpapi-dev' or newer.
Also, you get different PAPI versions for different releases and don't know if they have backward compats.

The Bionic Beaver (active development) papi trunk series
  5.6.0-1	release (universe)	2018-01-26
The Artful Aardvark (current stable release) papi trunk series
  5.5.1-2	release (universe)	2017-05-09
The Xenial Xerus (supported) papi trunk series
  5.4.3-2	release (universe)	2016-04-19
The Trusty Tahr (supported)
  5.3.0-3	release (universe)	2014-02-05